frontend/src/pages/Overview/reducer.js

```diff
@@ -119,10 +123,14 @@ export default function overviewReducer(state = defaultState, action) {
         .set("temporaryPermissions", fromJS(action.permissions));
     case PROJECT_NAME:
       return state.setIn(["projectToAdd", "displayName"], action.name);
-    case PROJECT_PROJECTED_BUDGET:
+    case ADD_PROJECT_PROJECTED_BUDGET:
       return state.merge({
-        projectToAdd: state.getIn(["projectToAdd"]).set("projectedBudgets", fromJS(action.projectedBudgets))
+        projectToAdd: state.get("projectToAdd").merge({
+          projectedBudgets: [...state.getIn(["projectToAdd", "projectedBudgets"]).toJS(), action.projectedBudget]
+        })
       });
+    case EDIT_PROJECT_PROJECTED_BUDGET:
+      return state.setIn(["projectToAdd", "projectedBudgets", action.index, "value"], action.budgetAmountEdit);
```

Comment:
I wouldn't use index here I think it's prone to error.
Suggestion:
- rename EDIT_PROJECT_PROJECTED_BUDGET to EDIT_PROJECT_PROJECTED_BUDGET_AMOUNT
- EDIT_PROJECT_PROJECTED_BUDGET_AMOUNT- pass a projectedBudget, you can find the old one by organization and currency